homeassistant/components/derivative/config_flow.py

```diff
@@ -31,49 +30,55 @@
 )
 
 UNIT_PREFIXES = [
-    {"value": "none", "label": "none"},
-    {"value": "n", "label": "n (nano)"},
-    {"value": "µ", "label": "µ (micro)"},
-    {"value": "m", "label": "m (milli)"},
-    {"value": "k", "label": "k (kilo)"},
-    {"value": "M", "label": "M (mega)"},
-    {"value": "G", "label": "T (tera)"},
```

Comment:
btw, these old `value`'s look wrong but would like confirmation that I have updated them correctly

Reply:
Oops, that was indeed wrong, sorry about it!